apps/src/utils.js

```diff
@@ -10,7 +11,7 @@ import constants from './constants';
  * @param subsequence Array - A subsequence of the given sequence.
  * @returns boolean - whether or not subsequence is really a subsequence of sequence.
  */
-export function isSubsequence(sequence, subsequence) {
+export function isSubsequence<T>(sequence:Array<T>, subsequence:Array<T>):bool {
```

Comment:
I've seen this somewhere before... oh yeah, it's just [AS3](http://help.adobe.com/en_US/ActionScript/3.0_ProgrammingAS3/WS5b3ccc516d4fbf351e63e3d118a9b90204-7f57.html).